src/AdvancementAPI.java

```diff
@@ -389,4 +389,81 @@ public String toString(){
     public String toString() {
         return "Advancement(" + id + "|" + this.title + ")";
     }
+
+
+    //BEGIN UTIL
+    protected static JSONObject convertItemToJSON(ItemStack item,JSONObject itemJSON){
+        itemJSON.put("item", "minecraft:"+ item.getType().name().toLowerCase());
+        itemJSON.put("amount", item.getAmount());
+        itemJSON.put("data",item.getData().getData());
+        return itemJSON;
+    }
+    //BEGIN CLASSES
+    public static class Condition{
+       protected String name;
+       protected Object set;
```

Comment:
It should be an object so the user can provide a String. Remember the JSON Object is a HashMap which accepts two objects